nixos/modules/services/x11/display-managers/auto.nix

```diff
@@ -41,28 +41,12 @@ in
 
   config = mkIf cfg.enable {
 
-    services.xserver.displayManager.lightdm = {
+    services.xserver.displayManager.slim = {
       enable = true;
-      autoLogin = {
-        enable = true;
-        user = cfg.user;
-      };
+      autoLogin = true;
+      defaultUser = cfg.user;
```

Comment:
Not an issue with this PR but we should standardize the settings so ```autoLogin``` works the same regardless of DM.